### PR TITLE
Use seed! instead of srand

### PR DESCRIPTION
**Pull request by andreasnoack**
We are trying to get as much to pass before tomorrow evening so a new release with this change would be appreciated.

### CODE REVIEW

**Comment by andreasnoack** [comment]
The failure on nightly is because the general Travis script needs adjustments which we are working on. Locally, the tests pass on lastest master.

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ os:
   - linux
   - osx
 julia:
+  - 0.7
```
Comment (ararslan) [inline .travis.yml:6]: Can you add 1.0 here as well?

Comment (andreasnoack) [reply .travis.yml:6]: Done

**Comment by andreasnoack** [comment]
@timholy Quite a few packages depend on this so it would be great to have a new version released with these updates.

**Comment by ararslan** [comment]
I found Tim in the hallway and convinced him to take a look at this. :)

**Pull request merged**